### PR TITLE
Add changelog for 1.0.0

### PR DESCRIPTION
**Pull request by rmarianski**
@nvkelso could you review please?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+v1.0.0
+------
+* **Release date**: 2016-10-04
+* **Requires:** [tileserver v1.0.0](https://github.com/mapzen/tileserver/releases/tag/v1.0.0) and [tilequeue v1.0.0](https://github.com/mapzen/tilequeue/releases/tag/v1.0.0) and [mapbox-vector-tile 1.0.0](https://pypi.python.org/pypi/mapbox-vector-tile/1.0.0).
+* Guard against intersecting with same ids during admin boundary processing.
+* Rank only features within the unpadded bounds of the tile. Drop unranked features only within that range too.
+* Drop linear boundaries, as these break the admin boundary processing code. Find alternatives for the test cases depending on linear boundaries.
+* Add pyclipper dependency to requirements.
+* Include name:short as a tag name alternate.
+
```
Comment (nvkelso) [inline CHANGELOG.md]: Additions:

* Fixed bug to restore some missing low-zoom region boundary lines.
* Fixed bug to fully enable new map_unit boundary lines at low-zooms.
* Low-zoom boundary lines now have custom min_zoom values.
* All features in place layer now have custom min_zoom values.
* Update data query to adapt to upstream OpenStreetMap healthcare speciality bulk edit.
* Fixed typo for `protction_title` to `protection_title` for National Forest features in `pois` layers.
* [docs] Cleanup docs generally, clarify relationship between `pois` and `landuse` layers, and remove promise about `tier` property (which will probably be deprecated).
* [docs] Migrate docs to reference generic Mapzen API keys.

**Comment by rmarianski** [comment]
All makes sense. Updated in 9a8e0e7.

**Comment by nvkelso** [comment]
Bumped docs version and clarifed feature names and localized names in https://github.com/tilezen/vector-datasource/pull/1099/commits/87b9ab6e272e7bc50de4c590cd3c1d240fd44a2d.

/cc @bcamper

**Comment by nvkelso** [comment]
Here's precog preview of v1 docs:

- https://precog.mapzen.com/mapzen/mapzen-docs-generator/nvkelso/vector-tiles-v1/documentation/vector-tiles/

With PR to merge that (only once v1 goes live on prod): 

- https://github.com/mapzen/mapzen-docs-generator/pull/204

**Pull request merged**